### ISSUE TITLE
Missing **bias** in DCGAN notebook in **conv()**

### ISSUE THREAD

**Issue by hpkhanh1610** [udacity/deep-learning-v2-pytorch#47]
Hi,

In the function **conv()** in DCGAN notebook, when defining the **conv_layer**, by default you create a **conv_layer** with no bias no matter with or without **batch norm**. I think you should put that **conv_layer** in **if-else** statement so that we have the bias in the conv_layer when we don't use batch norm.
```if batch_norm: conv_layer = ...**bias=False** else **bias=True**```

**Comment by cezannec** [comment]
In this case, best practice is to have bias set to False for the convolutional layers as well. This choice was informed by the original paper and example implementation within the PyTorch repo: https://github.com/pytorch/examples/blob/master/dcgan/main.py
